src/aframe-streetmix-parsers.js

```diff
@@ -841,7 +859,7 @@ function processSegments (segments, showStriping, length) {
       let reusableObjectStencilsParentEl;
 
       groundMixinId = 'bright-lane';
-      segmentParentEl.append(createDriveLaneElement([variantList[0], 'car'], positionX, segmentWidthInMeters, length));
+      segmentParentEl.append(createDriveLaneElement([variantList[0], 'car'], positionX, segmentWidthInMeters, length, false, showVehicles));
```

Comment:
looks like the `false` should be removed here? was probably used in testing? should be this instead:
      segmentParentEl.append(createDriveLaneElement([variantList[0], 'car'], positionX, segmentWidthInMeters, length, showVehicles));